### ISSUE TITLE
Make editor persistence optional

### ISSUE THREAD

**Issue by fregante** [pixiebrix/pixiebrix-extension#4923]
The editor is a little slow for me, would it be possible to optionally disable it completely via "Advanced" flag in the options at least until we fix its performance issues? cc @BALEHOK 

### Related

- https://github.com/pixiebrix/pixiebrix-extension/issues/4898

**Comment by twschiller** [comment]
@fregante the performance is a known issue

There's some tickets we're planning on addressing this week prior to 1.7.16 release:
- https://github.com/pixiebrix/pixiebrix-extension/issues/4898

**Comment by fregante** [comment]
Yes, but that's not going to happen overnight, whereas a (temporary)? toggle could. I'm seeing several-second lags and it's making work slower

**Comment by twschiller** [comment]
If you switch this line to just the editorSlice.reducer, I think you should be OK: https://github.com/pixiebrix/pixiebrix-extension/blob/3589ad20010142401557b7be0cf4e4b5142204e3/src/pageEditor/store.ts#L83-L83

@BALEHOK and @BLoe should be looking into performance today. I kicked up a conversation on status here: https://pixiebrix.slack.com/archives/C042LH845RV/p1672749035013239

If the persistence is the cause and we don't have a fix, we'll turn off the persistence by default for everyone
